app/Resources/FOSUserBundle/serializer/Model.User.yml

```diff
@@ -0,0 +1,11 @@
+FOS\UserBundle\Model\User:
```

Comment:
I think this file should go to `app/Resources/FOSUserBundle/config/serializer/Model.User.yml`.